documentation/source/user_section/tutorials/other/spinalcord-smoothing.rst

```diff
@@ -0,0 +1,34 @@
+.. TODO:
+
+   Is this one-page tutorial necessary? It is basically just telling users that the ``sct_smooth_spinalcord`` tool exists. (Compared to other tutorials, which demonstrate multi-step workflows.)
+
+   So, I am thinking that maybe this page will be unnecessary once we organize the "Command-Line Tools" page into one-page-per-script. We could simply have all of this information on the dedicated "sct_smooth_spinalcord" page instead, and save the "Tutorials" for complex workflows only.
+
+.. _spinalcord-smoothing:
+
+Spinal cord smoothing as a preprocessing operation
+##################################################
+
+SCT provides a spinal cord smoothing function that takes into account the curvature of the cord, preserving the boundary between the spinal cord and cerebrospinal fluid. This is useful in a variety of situations:
+
+* It can be used to improve sensitivity of fMRI results while minimizing contamination from CSF
+* It can also be used to obtain more reliable cord segmentations, because smoothing will sharpen the edge of the cord and will blur out possible artifacts at the cord/CSF interface.
+
+.. code::
+
+   sct_smooth_spinalcord -i t1.nii.gz -s t1_seg.nii.gz
```

Comment:
Adding to this. It would be nice to specify where the data is . Is it in `/single_subject/data/t1`. I think it till needs `t1_seg.nii.gz` . So far I got the following

<details>
<summary>Output</summary>

```
➜  data git:(sy/test-sct-tuts) ✗ ls
dmri  fmri  LICENSE.txt  mt  README.txt  t1  t2  t2s
➜  data git:(sy/test-sct-tuts) ✗ cd t1
➜  t1 git:(sy/test-sct-tuts) ✗ ls
t1.nii.gz
➜  t1 git:(sy/test-sct-tuts) ✗ sct_smooth_spinalcord -i t1.nii.gz -s t1_seg.nii.gz

--
Spinal Cord Toolbox (git-sy/test-sct-tuts-30e649037ab929937944d0df064b7a5cee32fb04)

sct_smooth_spinalcord -i t1.nii.gz -s t1_seg.nii.gz
--


Check input arguments...
  Volume to smooth .................. t1.nii.gz
  Centerline ........................ t1_seg.nii.gz
  Sigma (mm) ........................ [0, 0, 3]
  Verbose ........................... 1
Creating temporary folder (/tmp/sct-20211003164127.554537-smooth_spinalcord-4h7ensjk)

Copying input data to tmp folder and convert to nii...
cp t1.nii.gz /tmp/sct-20211003164127.554537-smooth_spinalcord-4h7ensjk/anat.nii.gz
Traceback (most recent call last):
  File "/home/srishtiy/dev/neuropoly/spinalcordtoolbox/spinalcordtoolbox/scripts/sct_smooth_spinalcord.py", line 269, in <module>
    main(sys.argv[1:])
  File "/home/srishtiy/dev/neuropoly/spinalcordtoolbox/spinalcordtoolbox/scripts/sct_smooth_spinalcord.py", line 171, in main
    copy(fname_centerline, os.path.join(path_tmp, "centerline" + ext_centerline))
  File "/home/srishtiy/dev/neuropoly/spinalcordtoolbox/spinalcordtoolbox/utils/fs.py", line 238, in copy
    contents = os.listdir(folder)
```

</details>